service/src/main/java/org/cbioportal/service/exception/PatientNotFoundException.java

```diff
@@ -2,13 +2,23 @@
 
 public class PatientNotFoundException extends Exception {
```

Comment:
this now functions as "Patient Not Found In Study" exception (rather than global search) .. or maybe it handles both cases.